lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -163,7 +163,7 @@ function findDifference(path, actual, expected) {
     const expectedValue = expected[key];
 
     if (!(key in actual)) {
-      return {keyPath, undefined, expectedValue};
+      return {path: keyPath, actual: undefined, expected: expectedValue};
```

Comment:
whoops, well that won't work.

(`{undefined}` alone seems like a ridiculous thing that should be caught instead of giving `{undefined: undefined}` :)